### ISSUE TITLE
Mining program unexpected exit.

### ISSUE THREAD

**Issue by jmesfo0** [NebuTech/NBMiner#569]
0x00000001400E38A8 (0x000000000E69D310 0x000000000431B5E0 0x0000000000000000 0x000000000431C700), () + 0xE38A8 bytes(s)
0x00000001400E6ED6 (0x000000000431B8E0 0x000000000E7D59F0 0x00000000028EA8E0 0x000000000431C700), () + 0xE6ED6 bytes(s)
0x000000014010E077 (0x0000000000253D8C 0x000000000025877F 0xFFFFFFFFFFFFFFFE 0x001F0000000A07E5), () + 0x10E077 bytes(s)
0x00000001400ABB25 (0x0000017CD4D9B55A 0x000000000431C8E0 0x0000000000000000 0x0000017CD4D9A82F), () + 0xABB25 bytes(s)
0x00000001400CEC91 (0x000000000E771CE0 0x0000017CD4D9B55A 0x00000000000000A4 0x0000000000000000), () + 0xCEC91 bytes(s)
0x000000014032A6E3 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), () + 0x32A6E3 bytes(s)
0x0000000140310139 (0x00000000035806B0 0x000000000357EB30 0x000000000E771CE0 0x0000000000000001), () + 0x310139 bytes(s)
0x000000014030FE47 (0x000000000357EB30 0x000000000357EB30 0x0000000000000000 0x000000000E771CE0), () + 0x30FE47 bytes(s)
0x000000014030FF09 (0x00004E7224F68095 0x0000000000000000 0x000000000357EB30 0x0000000000000000), () + 0x30FF09 bytes(s)
0x00000001403118BE (0x000000000357EB30 0x0000000000000000 0x0000000000000000 0x0000000000000000), () + 0x3118BE bytes(s)
0x00000001403FCC50 (0x00000000028E24A0 0x0000000000000000 0x000000000E7E6340 0x0000000000000000), () + 0x3FCC50 bytes(s)
0x00000001403BABEB (0x000000000357C938 0x000000000357C938 0x00000000028E2020 0x000000000357C8E0), () + 0x3BABEB bytes(s)
0x00000001403659B1 (0x000000000357C938 0x000000000357C938 0x0000000000000057 0x000000000357C938), () + 0x3659B1 bytes(s)
0x0000000140366C7C (0x000000000E758630 0x000000000E758630 0x0000000000000000 0x0000000000000000), () + 0x366C7C bytes(s)
0x0000000140568750 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), () + 0x568750 bytes(s)
0x00007FFAD1AC7034 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), BaseThreadInitThunk() + 0x14 bytes(s)
0x00007FFAD1C02651 (0x0000000000000000 0x0000000000000000 0x0000000000000000 0x0000000000000000), RtlUserThreadStart() + 0x21 bytes(s)
[22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[22:37:09] ERROR - Mining program unexpected exit.
[22:37:09] ERROR - Code: -1073741676, Reason: Process crashed
[22:37:09] ERROR - Restart miner after 10 secs ...
[22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by rtpraul** [comment]
Same here twice with two different cards in less than 10 minutes. 3060ti

**Comment by rtpraul** [comment]
> Same here twice with two different cards in less than 10 minutes. 3060ti

and again, third time in 10 minutes. Back to 39.6

**Comment by rtpraul** [comment]
> > Same here twice with two different cards in less than 10 minutes. 3060ti
> 
> and again, third time in 10 minutes. Back to 39.6

Solved it rebooting system, since with the 39.6 was happening as well.
By the moment 13 minutes with no miner restarts.

**Comment by rtpraul** [comment]
Same here, already several times.

**Comment by rtpraul** [comment]
More people facing the same issue.
Constant miner crashes.

https://www.reddit.com/r/EtherMining/comments/qjhk7f/nbminer_397_was_released/

**Comment by jmesfo0** [comment]
I ended up going back to trex miner until this gets fixed. Previous versions of NBMiner constantly "auto tune" my LHR cards when I can simply just lock it in at 74 or 72 or 70 with trex miner --lhr-autotune-mode off.

**Comment by magpern** [comment]
I too can confirm 39.7 crashes a lot in HiveOS. Illegal instruction, lost GPU etc.
Reverting back to 39.6 and everything is stable. Same OC settings.

As for hashrate. I get the exact same hashrates in 39.6 and 39.7, 39.7 is just more unstable

**Comment by rtpraul** [comment]
In my case I had to lower memory clock for the 5* 3060ti, if not, it crashed every two minutes. Driver 470.74
The previous OCs were just fine with version 39.6.
But went back to 39.6 and it happened again and again. The issue started after updating hiveOs with NB 39.7

So now it has been stable for two days on 39.7 with lower memory clock.
What I did troubleshooting:

Back to 39.6, driver 470.74 - same.
Reinstalled hiveOs on different USB - same
Update driver to 470.82 - same
Finally lower OC and now stable but a bit less Hashrate: driver 470.82 over 39.7

.

**Comment by magpern** [comment]
@rtpraul I noticed cards where set in mode P0 on the upgrade, and that kills cards overclocked for P2. Now I changed the OC to fit P0 mode (lowered everything by 400)
But, like you, I also have trouble with my 3060ti cards, not the 3060

**Comment by rtpraul** [comment]
True, 3060 like a charm, more Hashrate, same OC. Happy with them over Nbminer, that's true.

Thanks for the Infos, I'll check it

**Comment by giosal** [comment]
Same happening here on 3080 Mobile using drivers 496.27 and NBMiner 40.1 Win

**Comment by luy-cyber** [comment]
O mesmo acontecendo aqui, placa de video 1650

**Comment by luy-cyber** [comment]
Same happening here, 1650 video card
